build.sh

```diff
@@ -80,6 +80,7 @@ usage() {
 Usage: $0 <options>
   Optional options:
      --be               build Backend
+     --format-lib       build StarRocks format library
```

Comment:
need provide some more info about this library, e.g. only works with shared-data mode cluster.

Reply:
OK